Misc/NEWS.d/next/Core_and_Builtins/2025-09-24-17-08-42.gh-issue-133059.EXvxb7.rst

```diff
@@ -0,0 +1 @@
+Increased the number of cached small positive integers from 256 to 1024.
```

Comment:
This sentence is incorrect: the *number* of cached integers changed from 260 to 1030, no?  It might be clearer to state the old range and the new range.